esp32_s3_usb_otg/include/bsp/esp32_s3_usb_otg.h

```diff
@@ -126,7 +126,11 @@ esp_err_t bsp_sdcard_unmount(void);
 #define BSP_LCD_V_RES              (240)
 #define BSP_LCD_PIXEL_CLOCK_HZ     (40 * 1000 * 1000)
 #define BSP_LCD_SPI_NUM            (SPI3_HOST)
-#define BSP_LCD_BUFF_SIZE          (BSP_LCD_H_RES * 30) // Frame buffer size in pixels
+#define BSP_LCD_BITS_PER_PIXEL     (16)
+#define BSP_LCD_COLOR_SPACE        (ESP_LCD_COLOR_SPACE_RGB)
```

Comment:
Could we move these definitions to `display.h`? (color space and bits per pixel)

Reply:
Yes, I will do it.